tests/e2e/pageObjects/workbench-page.ts

```diff
@@ -170,8 +172,8 @@ export class WorkbenchPage {
   * @param command The command
   * @param speed The speed in seconds. Default is 1
   */
-  async sendCommandInWorkbench(command: string, speed = 1): Promise<void>{
-      await t.typeText(this.queryInput, command, { replace: true, speed: speed});
+  async sendCommandInWorkbench(command: string, speed = 1, paste = false): Promise<void>{
+      await t.typeText(this.queryInput, command, { replace: true, speed: speed, paste: paste});
```

Comment:
Do not need to duplicate key/value if they are the same.
Could you please simplify to `{ replace: true, speed, paste }`

Reply:
done